README.md

```diff
@@ -64,6 +64,34 @@ Please install it and then accept the invitation received on your email address.
 
 The application can be downloaded from here: [Hubstaff Tracker](https://app.hubstaff.com/download)
 
+#### INTERNAL projects
+Below you can find the list of INTERNAL projects alongside with their descriptions.
+Descriptions tell you where you should bill your INTERNAL time on hubstaff.
+
+| Project                                          | Description                                                                                                                                           |
+| -----------------------------------------------  | ----------------------------------------------------------------------------------------------------------------------------------------------------  |
+| INTERNAL / content marketing                     | writing articles or blog posts under company name; presenting company in the outside world;                                                           |
+| INTERNAL / django application dockerization      | modifying our internal django docker template project;                                                                                                |
+| INTERNAL / finding new partners                  | time spent on finding new partners/customers;                                                                                                         |
+| INTERNAL / internal infrastructure management    | anything related to preparing your workspace to work; creating encrypted partitions, installing vms or any software needed to work;                   |
```

Comment:
... which is not needed to any specific project. If you are setting up environment specific to a project or customer (internal or not), bill the time you spend on it on that project.